contracts/staking/test/unit_tests/stake_test.ts

```diff
@@ -373,18 +373,15 @@ blockchainTests.resets('MixinStake unit tests', env => {
             expect(increaseNextBalanceEvents).to.be.length(0);
         });
 
-        it('moves the owner stake between the same pointer when both are undelegated', async () => {
+        it('does nothing when moving the owner stake from undelegated to undelegated', async () => {
```

Comment:
A test for the noop if amount == 0 would be nice

Reply:
Lol was literally just adding that 😆